controller_manager:
  ros__parameters:
    update_rate: 1000 # Hz

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController


joint_trajectory_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
